mw-webapp/src/logic/waysTable/renderLinkInCell/renderLinkInCell.tsx

```diff
@@ -8,6 +9,7 @@ export const renderLinkInCell = (path: string, value: string) => {
     <Link
       value={value}
```

Comment:
What do you think about using Link component in columns file without using additional function that just return Link component?

Reply:
It's a little bit more encapsulated right now, but yes we can